message ServerInfo_Replay {
	optional sint32 game_id = 1 [default = -1];
	optional string room_name = 2;
	optional uint32 time_started = 3;
	optional uint32 length = 4;
	optional string game_name = 5;
	optional string replay_name = 6;
	repeated string player_names = 7;
}
